### DISCUSSION TITLE
16.08.21 backup

### DISCUSSION THREAD

**Discussion by AtFreezingPoint** [Ludeon/RimWorld-Turkish#327] [Ideas]
[Backup](https://github.com/AtFreezingPoint/RimWorld-Turkish)
16.08.2021 tarihli proje yedeği, herhangi bir sıkıntı çıkarsa veya eski versiyondan herhangi birşeye ihtiyaç olunursa diye haftada bir filan aktif olan herkesin açmasını rica ediyorum. Fork açmak kesinlikle eski bir versiyona erişimi çok daha kolaylaştırıyor.

**Comment by LadySmith** [comment]
[Bu şekilde](https://github.com/Ludeon/RimWorld-Turkish/tree/9c8dbf2c293395ec081a3c3da8ec261584decfcb) atmak istediniz herhalde. History üzerinden geri dönmek çok kolay. Referans olarak, kırmızıya tıklayarak geçmişteki herhangi bir commite bakabilirsiniz..
![red](https://user-images.githubusercontent.com/2457303/129774617-c16076fc-8dca-4858-8ede-f9cd46bcf1a9.png)

**Reply by AtFreezingPoint** [reply]
Öyle olunca büyük değişimler yapılınca sorun çıkarsa revert atıp yapmak github uygulamasından çok zor oluyor aralarında geçiş yapamadığın için hızlı bir şekilde.

**Reply by LadySmith** [reply]
Tabii öyle de olur fakat ana sayfadan buraya gelmek 2-3 click sürüyor. Mesela `dd.mm.yy` diye branch açsak daha kolay olmaz mı?

**Reply by AtFreezingPoint** [reply]
Branch daha iyi olur ayn dün aklıma gelmedi
